src/elm/View/Posts.elm

```diff
@@ -10,7 +10,7 @@ import Msg exposing (Msg)
 import Time
 
 
-view : Data.SectionId -> List Data.Post -> Html.Html Msg
+view : Data.SectionId -> List Data.Post -> Html.Html msg
```

Comment:
Why is the Posts view part of this work?

Reply:
the draggable pile works with a defined `Msg` so I need to either pass in views that use that  or  are generic.

Reply:
I hadn't realised that `Post`s should be draggable as well as `Image`s at this point.